modules/testing-3-800.adoc

```diff
@@ -0,0 +1,759 @@
+
+.Procedure 
+
+. On the *Repositories* page of the {productname} 3.8 beta UI, click the name of the image you want to delete, for example, `quay/admin/busybox`. 
+
+. Click the *More Actions* drop-down menu. 
+
+. Click *Delete*. 
++
+[NOTE]
+====
+If desired, you could click *Make Public* or *Make Private*. 
+====
+
+. Type *confirm* in the box, and then click *Delete*. 
+
+. After deletion, you are returned to the *Repositories* page. 
+
+=== Enabling the {productname} legacy UI
+
+. In the navigation pane of your {productname} deployment, you are given the option to toggle between *Current UI* and *New UI*. Click the toggle button to set it to *Current UI*. 
++
+image:38-ui-toggle.png[{productname} 3.8 UI toggle]
+
+[[leveraging-storage-quota-limits]]
+
+=== Leveraging storage quota limits 
+
+With {productname} 3.8, the proxy cache feature has been enhanced with an auto-pruning feature for tagged images. The auto-pruning of image tags is only available when a proxied namespace has quota limitations configured. Currently, if an image size is greater than quota for an organization, the image is skipped from being uploaded until an administrator creates the necessary space. Now, when an image is pushed that exceeds the allotted space, the auto-pruning enhancement marks the least recently used tags for deletion. As a result, the new image tag is stored, while the least used image tag is marked for deletion. 
+
+=== Testing storage quota limits auto-pruning feature 
+
+Use the following procedure to test the auto-pruning feature of an organization with proxy cache and storage quota limitations enabled.
+
+.Prerequisites 
+
+* Your organization is configured to serve as a proxy organization. The following example proxies from quay.io. 
+
+* `FEATURE_PROXY_CACHE` is set to `true` in your `config.yaml` file. 
+
+* `FEATURE_QUOTA_MANAGEMENT` is set to `true` in your `config.yaml` file. 
+
+* Your organization is configured with a quota limit, for example, `150 MB`. 
+
+.Procedure 
+
+. Pull an image to your repository from your proxy organization, for example:
++
+----
+$ podman pull quay-server.example.com/proxytest/projectquay/clair:4.2.3
+----
+
+. Depending on the space left in your repository, you might need to pull additional images from your proxy organization, for example:
++
+----
+$ podman pull quay-server.example.com/proxytest/projectquay/clair:4.1.5
+----
+
+. In the {productname} registry UI, click the name of your repository.
+
+* Click *Tags* in the navigation pane and ensure that `clair:4.2.3` and `clair:4.1.5` are tagged. 
+
+. Pull the last image that will result in your repository exceeding the the allotted quota, for example:
++
+----
+$ podman pull quay-server.example.com/proxytest/projectquay/clair:4.1.4
+----
+
+. Refresh the *Tags* page of your {productname} registry. The first pushed that you pushed, for example, `clair:4.2.3` should have been auto-pruned. The *Tags* page should now show `clair:4.2.3` and `clair:4.1.4`. 
```

Comment:
typo: `The first pushed that you pushed` should be `The first clair tag 4.2.3 that you pushed`  or just `the first image that you pushed`?